Formula/l/luanti.rb

```diff
@@ -0,0 +1,79 @@
+class Luanti < Formula
+  desc "Free, open source voxel game engine and game"
+  homepage "https://www.luanti.org"
+  license "LGPL-2.1-or-later"
+
+  stable do
+    url "https://github.com/luanti-org/luanti/archive/refs/tags/5.11.0.tar.gz"
+    sha256 "70e531d0776988ce6e579ea5490fdf6be3e349a4ade5281f5111aa4fdd8ee510"
+
+    resource "minetest_game" do
+      url "https://github.com/luanti-org/minetest_game/archive/refs/tags/5.8.0.tar.gz"
+      sha256 "33a3bb43b08497a0bdb2f49f140a2829e582d5c16c0ad52be1595c803f706912"
+    end
+  end
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  head do
+    url "https://github.com/luanti-org/luanti.git", branch: "master"
+
+    resource "minetest_game" do
+      url "https://github.com/luanti-org/minetest_game.git", branch: "master"
+    end
+  end
+
+  depends_on "cmake" => :build
+  depends_on "curl"
+  depends_on "freetype"
+  depends_on "gettext"
+  depends_on "gmp"
+  depends_on "jpeg-turbo"
+  depends_on "jsoncpp"
+  depends_on "libogg"
+  depends_on "libpng"
+  depends_on "libvorbis"
+  depends_on "libx11"
+  depends_on "libxi"
+  depends_on "libxxf86vm"
+  depends_on :linux
```

Comment:
Is there a way for this to be disabled on macos with a suggestion to install the cask? I'm not sure the `replacement` parameter for `disable!` can point specifically to a cask.

Reply:
Not really. We don't use `disabled` for that, that's why we have `:linux` and `:macos`

I you try to install `luanti` on Mac, it will tell you:
```
brew install luanti
Warning: Treating luanti as a formula. For the cask, use homebrew/cask/luanti or specify the `--cask` flag. To silence this message, use the `--formula` flag.
luanti: Linux is required for this software.
```

I think that's enough information to not get lost, users should be able to find out what to do.
It's not perfect but it should be ok.